pkg/platform/microservice/configFiles/service.go

```diff
@@ -80,56 +90,64 @@ func (s *service) UpdateConfigFiles(w http.ResponseWriter, r *http.Request) {
 	allowed := s.k8sDolittleRepo.CanModifyApplicationWithResponse(w, customerID, applicationID, userID)
 
 	if !allowed {
+		logContext.Info("UpdateConfigFiles: not allowed ")
+
 		return
 	}
 
 	if file == nil {
-		errMsg := "UpdateConfigFiles ERROR: No file"
-		fmt.Println(errMsg)
-		utils.RespondWithError(w, http.StatusBadRequest, errMsg)
+		msg := "UpdateConfigFiles ERROR: No file"
+
+		logContext.Info(msg)
+
+		utils.RespondWithError(w, http.StatusBadRequest, msg)
 		return
 	}
 
 	if strings.TrimSpace(handler.Filename) != handler.Filename {
-		errMsg := "UpdateEnvironmentVariables ERROR: No spaces allowed in config file name"
-		fmt.Println(errMsg)
-		utils.RespondWithError(w, http.StatusBadRequest, errMsg)
-		return	
-	}
+		msg := "UpdateEnvironmentVariables ERROR: No spaces allowed in config file name"
+
+		logContext.Info(msg)
 
+		utils.RespondWithError(w, http.StatusBadRequest, msg)
+		return
+	}
 
 	// file size limit from header.Size()
 	if handler.Size > 3145728 {
-		errMsg := "UpdateConfigFiles ERROR: File size too large"
-		fmt.Println(errMsg)
-		utils.RespondWithError(w, http.StatusBadRequest, errMsg)
+		msg := "UpdateConfigFiles ERROR: File size too large"
+
+		logContext.Info(msg)
+
+		utils.RespondWithError(w, http.StatusBadRequest, msg)
 		return
 	}
 
 	s.logContext.Info("Update config files")
 
-	var input platform.StudioConfigFile
-
+	
 	body, err := ioutil.ReadAll(file)
-
+	
 	if err != nil {
-		fmt.Println("UpdateConfigFiles ERROR: " + err.Error())
-		utils.RespondWithError(w, http.StatusBadRequest, "Invalid request payload")
+		msg := "UpdateConfigFiles ERROR: Invalid file"
+
+		logContext.Info(msg)
+
+		utils.RespondWithError(w, http.StatusBadRequest, "Invalid file")
```

Comment:
```suggestion
		utils.RespondWithError(w, http.StatusBadRequest, msg)
```

I assume you meant to use the `msg` variable here in the response.